src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4571,6 +4565,12 @@ public override IOperation VisitTypeOf(ITypeOfOperation operation, int? captureI
             return new TypeOfExpression(operation.TypeOperand, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        public override IOperation VisitEmpty(IEmptyOperation operation, int? captureIdForResult)
```

Comment:
>public override IOperation VisitEmpty(IEmptyOperation operation, int? captureIdForResult) [](start = 8, length = 89)

I guess the method didn't have to be moved because it wasn't in the "not-implemented" region. But I guess that is fine.

Reply:
You're right.  I went ahead and removed this change from the file.